### PR TITLE
psycopg2cffi Tests

### PR DESCRIPTION
**Pull request by TimPansino**
Sidekick: @carrieedwards 

# Overview

* Import psycopg2cffi tests from internal repo.
* Add namespacing to tests to fix concurrency issues.

### CODE REVIEW

File: tests/testing_support/fixtures.py [hunk 1]
```diff
@@ -2055,6 +2055,22 @@ def _validate_transaction_error_trace_count(wrapped, instance, args,
     return _validate_transaction_error_trace_count
 
 
+def validate_transaction_slow_sql_count(num_slow_sql):
```
Comment (carrieedwards) [inline tests/testing_support/fixtures.py:2058]: Is this validator one we should also put in its own file to clear up fixtures.py?

Comment (TimPansino) [reply tests/testing_support/fixtures.py:2058]: Yep, good call.

**Review by carrieedwards — APPROVED** [review]
Looks good to me!

**Review by carrieedwards — COMMENTED** [review]
Looks good to me!

**Pull request merged**